cognite/neat/core/extractors/rules_to_graph_capturing_sheet.py

```diff
@@ -0,0 +1,153 @@
+import logging
+import uuid
+import warnings
+from pathlib import Path
+
+import xlsxwriter
+
+from cognite.neat.core.data_classes.transformation_rules import TransformationRules
+from cognite.neat.core.extractors.rules_to_graphql import repair_name as to_graphql_name
+
+
+def _get_column(column_number: int) -> str:
```

Comment:
This seems like a standard operation. Is there no such method in the excel packages?

Reply:
there is 👍 , implemented